docs/spec/spec.md

```diff
@@ -3,13 +3,17 @@
 _Owners_: @daneshk @kalaiyarasiganeshalingam  
 _Reviewers_: @daneshk  
 _Created_: 2021/12/10   
-_Updated_: 2021/12/10  
+_Updated_: 2022/02/10  
```

Comment:
We need to include the edition as well.

https://github.com/ballerina-platform/module-ballerina-http/blob/master/docs/spec/spec.md

Reply:
Fixed in [3d0a27f](https://github.com/ballerina-platform/module-ballerina-file/pull/417/commits/3d0a27ff8b3d2e26b3c212989836f6d26373a87e)